daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/DFDL_part2_attributes.xsd

```diff
@@ -789,6 +789,8 @@
       type="dfdl:YesNoEnum" />
     <xsd:attribute form="qualified" name="textBidiNumeralShapes"
       type="dfdl:TextBidiNumeralShapesEnum" />
+    <xsd:attribute form="qualified" name="textBidi"
+                   type="dfdl:YesNoEnum" />
```

Comment:
This change, and other textbidi changes, are in the wrong commit. This is a textBidi change and so should be in the first commit. This is another downside with including two separate commits/changes in the same pull request--it's easy to accidentally put things in the wrong commit.